contributed_definitions/nyaml/NXfit_region.yaml

```diff
@@ -0,0 +1,92 @@
+category: base
+doc: |
+  Description of region for peak fitting (i.e., in an NXpeak_model).
+  
+  Used for descibring a subpart of the abscissa (in 2D spectra) that is
+  used for fitting.
+type: group
+NXfit_region(NXobject):
+  label(NX_CHAR):
+    doc: |
+      Human-readable label which specifies which concept/entity
+      the region represents/identifies.
+  start(NX_NUMBER):
+    unit: NX_ANY
+    doc: |
+      Minimal value of the defined region.
+  stop(NX_NUMBER):
+    unit: NX_ANY
+    doc: |
+      Minimal value of the defined region.
+  (NXbackground):
+    doc: |
+      Fitted background (functional, position, and intensities).
```

Comment:
Also not sure I understand why a region should have a background. The fit might include one, the region rather not, no?